File: ProjectUtils/src/test/scala/com/mine/test/bean/Customer.scala
package com.mine.test.bean


class Customer {
    var id: String = _
    var cTime: String = _
    var uTime: String = _
    var birthDate: String = _
    var sex: Int = _
    var city: String = _
    var expectation: Int = _
    var province: String = _
    var sourceChannel: Int = _

    override def toString: String = {
        "id = " + id + "\tcTime = " + cTime + "\tuTime = " + uTime + "\tbirthDate = " + birthDate + "\tsex = " + sex + "\tcity = " + city + "\texpectation = " + expectation + "\tprovince = " + province + "\tsourceChannel = " + sourceChannel
    }
}

File: ProjectUtils/src/test/java/cn/mine/Customer.java
package cn.mine;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author ximing.wei 2021-06-15 16:45:53
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Customer {
    private String id;
    private String cTime;
    private String uTime;
    private String birthDate;
    private Integer sex;
    private String city;
    private Integer expectation;
    private String province;
    private Integer sourceChannel;

    @Override
    public String toString() {
        return "Customer{" +
                "id='" + id + '\'' +
                ", cTime='" + cTime + '\'' +
                ", uTime='" + uTime + '\'' +
                ", birthDate='" + birthDate + '\'' +
                ", sex=" + sex +
                ", city='" + city + '\'' +
                ", expectation=" + expectation +
                ", province='" + province + '\'' +
                ", sourceChannel=" + sourceChannel +
                '}';
    }
}
